Sources/Pokepay/BankAPI/User/BankPayTopUp.swift

```diff
@@ -7,12 +7,14 @@ public extension BankAPI.User {
         public let bankId: String
         public let amount: String
         public typealias Response = UserTransaction
+        public let requestId: String
         
-        public init(id: String, accountId: String , bankId: String, amount: String) {
+        public init(id: String, accountId: String , bankId: String, amount: String, requestId: String) {
```

Comment:
`requestId: String` should be `requestId: String? = nil` since it is optional.

Reply:
Thanks, I've fixed now.